### ISSUE TITLE
Lack of clear focus indicators and focusable elements on the Notification page

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1781]
On https://cryptpad.fr/notifications/, elements don't have a clear focus indicator and other buttons are not focusable:

For example: In the **History** section, I can navigate through notifications using the Tab key, but the focus indicator is unclear, it's just a blue line. Additionally, the "Load More History" button is not accessible via keyboard.

![Image](https://github.com/user-attachments/assets/c1bf7e2f-f060-4db5-8362-c5665275790d)
